Project.toml

```diff
@@ -27,7 +27,7 @@ test = ["Base64", "DelimitedFiles", "Test"]
 
 [compat]
 julia = "1"
-ArnoldiMethod = "0.0.4"
-DataStructures = "0.17.9, 0.18"
-Inflate = "0.1.1"
-SimpleTraits = "0.9.1"
+ArnoldiMethod = "0.0"
+DataStructures = "0.17, 0.18"
+Inflate = "0.1"
+SimpleTraits = "0.9"
```

Comment:
Are SimpleTraits 0.9.0, Inflate 0.1.0, DataStructures 0.17.0-0.17.8 compatible?

Reply:
It's passing CI so I think we're ok.

Reply:
Also these upper bounds are not a guarantee that these packages can't hold *each other* back.  So, for example, if `Inflate` was stuck on DataStructures 0.17, LightGraphs would only ever install 0.17 despite what it says here (fortunately that's not the case anyway).